src/config.js

```diff
@@ -28,20 +28,25 @@ function getCoveragePaths() {
 }
 
 const jestConfig = {
-    rootDir: PROJECT_ROOT,
-    moduleFileExtensions: ['js', 'html'],
+    // Inherited from @lwc/jest-preset
+    moduleFileExtensions: jestPreset.moduleFileExtensions || ['ts', 'js', 'html'],
     testEnvironment: jestPreset.testEnvironment || 'jsdom',
     transform: {
-        '^.+\\.(js|html|css)$': require.resolve('@lwc/jest-transformer'),
+        ...jestPreset.transform,
+        '^.+\\.(js|ts|html|css)$': require.resolve('@lwc/jest-transformer'),
     },
+    setupFilesAfterEnv: jestPreset.setupFilesAfterEnv || [],
+    snapshotSerializers: jestPreset.snapshotSerializers || [
+        require.resolve('@lwc/jest-serializer'),
+    ],
```

Comment:
I'm not sure why we have the pattern of `jestPreset.prop || 'hard-coded value matching jestPreset'` when we depend on a version of `jestPreset` where it's always defined, but I kept the pattern.

I also considered changing this section to just `...jestPreset`, but that would add in [`testMatch` and `coveragePathIgnorePatterns`](https://github.com/salesforce/lwc-test/blob/5b1d20c177c33d5faf7135aa133f3c4a3856b0fa/packages/%40lwc/jest-preset/jest-preset.js#L18-L19), and I don't know whether we want to do that.